_data/settings.yml

```diff
@@ -6,5 +6,6 @@ author:
 menu:
 - {name: 'Home', url: ''}
 - {name: 'Meetings', url: 'meetings/'}
+- {name: 'Topics', url: 'topics/'}
 - {name: 'Code of Conduct', url: 'code-of-conduct/'}
 - {name: 'Hosting', url: 'hosting/'}
```

Comment:
Should we have the graph here too?

Reply:
I was thinking that the graph needs a bit of work to be ready for prime time; that said, exposing it might bring interest from others. I'll add it.